test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+﻿Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
+
+    $skipTest = [System.Management.Automation.Platform]::IsLinux -or [System.Management.Automation.Platform]::IsOSX
```

Comment:
Why are we skipping Linux and OSX? If there is something blocking this from working in Linux and OSX, then we should open an issue and investigate.